docs/vz330_mellow/kit_contents.md

```diff
@@ -248,7 +249,14 @@ The following parts are included in the kit:
 * Exhaust filtration / HEPA filter ([Amazon 1](https://amzn.to/3lISPxr) / [AliExpress 1](https://s.click.aliexpress.com/e/_ABbR3U))
 * 2x Front led bars ([AliExpress 1](https://s.click.aliexpress.com/e/_DcRL3RT))
 * Raspberry Pi 3/4/Zero ([Official distributors](https://www.raspberrypi.com/resellers/))
-* Thermal fuse 125°C ([AliExpress](https://s.click.aliexpress.com/e/_AtAOQN))
+
+## Mellow defects
+
+These parts are currently still missing from Mellow's Kit BOM, but should eventually become part of it.
+
+* 1x Thermal fuse 125°C ([AliExpress](https://s.click.aliexpress.com/e/_AtAOQN))
```

Comment:
This can also be removed, as it's included in the bed heater itself now. 

The self-sourcing entry is fine to leave in, as it's not in the scope of this pull request or issue. :) 
We can resolve that later; I can't resolve it while you're away because it depends on your PR.